client/rest-high-level/src/main/java/org/opensearch/client/RequestConverters.java

```diff
@@ -433,9 +436,16 @@ static void addSearchRequestParams(Params params, SearchRequest searchRequest) {
         params.putParam(RestSearchAction.TYPED_KEYS_PARAM, "true");
         params.withRouting(searchRequest.routing());
         params.withPreference(searchRequest.preference());
+        if (searchRequest.pointInTimeBuilder() == null) {
+            params.withIndicesOptions(searchRequest.indicesOptions());
+        }
         params.withIndicesOptions(searchRequest.indicesOptions());
         params.withSearchType(searchRequest.searchType().name().toLowerCase(Locale.ROOT));
-        params.putParam("ccs_minimize_roundtrips", Boolean.toString(searchRequest.isCcsMinimizeRoundtrips()));
+        if (searchRequest.pointInTimeBuilder() != null) {
+            params.putParam("ccs_minimize_roundtrips", "false");
```

Comment:
Could you please add comment here why `isCcsMinimizeRoundtrips` is not respected for PIT?

Reply:
https://github.com/opensearch-project/OpenSearch/blob/a469a3cbcb7a7294a81bf7e4122968c2a9b32ca4/server/src/main/java/org/opensearch/action/search/SearchResponseMerger.java#L241 -- search responses merge is not supported for point in time as pit id is not passed, so this is not respected for PIT - will add the same as comment.